troposphere/static/js/components/modals/volume/VolumeAttachModal.react.js

```diff
@@ -227,12 +228,14 @@ define(
             instances = this.state.instances,
             content;
 
-        if (!instances) {
+        var activeInstances = instances && instances.filter(i => i.is_active());
+
+        if (instances == null) {
           content = this.renderLoadingContent();
-        } else if (instances.length <= 0) {
-          content = this.renderAttachRulesContent();
+        } else if (activeInstances.length > 0) {
+          content = this.renderAttachVolumeContent(activeInstances);
         } else {
-          content = this.renderAttachVolumeContent(instances);
+          content = this.renderAttachRulesContent(); 
         }
 
         return (
```

Comment:
It seems to render this okay.  It might just be that the GitHub diff format doesn't support it (like other editors \[[1](https://discuss.atom.io/t/problem-with-apostrophes-in-jsx-code/21431/2),[2](https://github.com/allanhortle/JSX/issues/4),[3](https://github.com/mxw/vim-jsx/issues/46)\]. We could wrap is it in `{" ... "}` or use an HTML entity - https://facebook.github.io/react/docs/jsx-gotchas.html#html-entities (but yuck).